### ISSUE TITLE
about table 3

### ISSUE THREAD

**Issue by iris0329** [Chat-3D/Chat-3D-v2#14]
Hi, thanks for providing this work.

![image](https://github.com/Chat-3D/Chat-3D-v2/assets/26380036/69159b09-d588-4b33-afd6-a2fdd79c338c)

1. I found there is no discussion for table 3 in the paper.
2. I found the acc@0.25 on the ScanRefer dataset is 35.9, but I didn't find out this number on the ScanRefer official site. 
3. I just wonder why the results seem to not be compared with the ScanRefer baseline method.

Did I miss anything?

**Comment by ZzZZCHS** [comment]
1. Table 3 aims to show that our method reaches SOTA among 3D LLM methods.
2. We haven't uploaded the results to the official site, but we have released the entire training and evaluation pipeline (including the checkpoints) in this repository. You can try it out to check the performance.
3. I think the main reason is the lack of data for 3D-LLM alignment. For the success of 2D-LLMs, they utilized a large amount of training data for alignment. Also, the intricate nature of 3D scenes necessitates a more tailored design to learn spatial relationships effectively. In this work, we use a simple alignment architecture and only use data sourced from scannet, which is far from enough to train a robust 3D LLM. Future works are needed to solve these problems.

**Comment by iris0329** [comment]
Ah, I see. Thanks for your detailed reply and open-source, and I tried the code.

I still would like to know, in the [ScanRefer benchmark](https://kaldir.vc.in.tum.de/scanrefer_benchmark/)
![image](https://github.com/Chat-3D/Chat-3D-v2/assets/26380036/8fd89594-e3ee-41ee-a333-730f7dabf3c3)


but Table 3 shows the different numbers for the same method, e.g. ScanRefer
![image](https://github.com/Chat-3D/Chat-3D-v2/assets/26380036/37159f9a-41dc-4c10-ac22-88da2eb5cf64)

I am kind of confused in reading the table as to why this difference will happen

Would you share with me how to match the two tables?

**Comment by ZzZZCHS** [comment]
The ScanRefer Benchmark is based on ScanRefer's test set, while the results in our Table 3 is on validation set.

We use the results from [ViL3DRel](https://arxiv.org/abs/2211.09646)'s Table 8. You can also find the same results (37.3/24.3) in [InstanceRefer](https://arxiv.org/abs/2103.01128) (Table 1) and [MVT](https://arxiv.org/abs/2204.02174) (Table 3).
